src/head/script.js

```diff
@@ -260,16 +228,9 @@ export default (config, globalOneTrustEvent) => {
 			if (config.enableGTM && !optout) {
 				insertGTM();
 			}
-			if (config.algoliaConfig.enableAA && !optout) {
-				insertAlgoliaAnalytics();
-			}
 			if (config.enableHotjar && !optout) {
 				insertHotjar();
 			}
 		}
-
-		if (config.enableFB && !optout) {
-			insertFB();
```

Comment:
Are we going to be good with the specific fb events while relying on gtm to load the lib? I think they are guarded to check so prob so..

Reply:
Yeah, all the facebook events are guarded with the `fbLoaded` check or with a check to `window.fbq`